tutorials/convert_lit_models.md

```diff
@@ -6,21 +6,21 @@ We provide a helpful script to convert models Lit-GPT models back to their equiv
 
 ```sh
 python scripts/convert_lit_checkpoint.py \
-    --checkpoint_path path/to/litgpt/model.pth \
-    --output_path out/converted.ckpt \
-    --config_path path/to/litgpt/config.json
+    --checkpoint_path checkpoints/repo_id/lit_model.pth \
+    --output_path output_path/converted.pth \
+    --config_path checkpoints/repo_id/config.json
```

Comment:
I guess we could get rid of the `config_path` argument now

Reply:
Let's do that in the PR where we copy the relevant files when creating checkpoint files.